solr/core/src/java/org/apache/solr/filestore/DistribFileStore.java

```diff
@@ -217,7 +217,7 @@ private boolean fetchFileFromNodeAndPersist(String fromNode) {
     }
 
     boolean fetchFromAnyNode() {
-      ArrayList<String> l = coreContainer.getFileStoreAPI().shuffledNodes();
+      ArrayList<String> l = FileStoreUtils.shuffledNodes(coreContainer);
```

Comment:
i know you are worried about scope, but maybe not `l`?  `liveNodes`?

Reply:
or `nodes`?

Reply:
I went with `nodesToAttemptFetchFrom`